lib/src/utils/currency_utils.dart

```diff
@@ -3,10 +3,12 @@ import 'package:intl/intl.dart';
 
 class CurrencyUtil {
   bool isInternationalCompany;
+  bool companyUsesFrenchForCurrencyConversion;
```

Comment:
for now it's ok but in future let's make enums and deprecate international or other.